### DISCUSSION TITLE
Can you search availability by equipment type (e.g. RV 30) in Yellowstone campgrounds (not recreation.gov)

### DISCUSSION THREAD

**Discussion by ccabrich** [juftin/camply#224] [Q&A]
First of all Justin, what you have built is amazing and useful. It is so hard to get campgrounds these days...and camply just makes it so easy!

The question is: can you check availability by type of equipment for the (non recreation.gov) Yellowstone campgrounds?

Bonus question: can you check availability for two different (non consecutive) nights, or should I launch two separate searches?

Thanks!

**Comment by juftin** [comment]
Hey @ccabrich thank you for the kind words! Unfortunately searching by equipment isn't yet supported by the `Yellowstone` provider - it's probably doable, I think they allow for it on the site and return some data back about equipment. I'l try to look into that. 

For the bonus question, the answer is also no - unless the `--weekends` search functionality is what you're looking for. Is there a certain pattern you're looking for, like "just check Mondays"? The python API supports multiple search windows, it's probably time that I implement that for the CLI too.
